docs/data/joy/customization/theme-typography/theme-typography.md

```diff
@@ -0,0 +1,161 @@
+# Theme typography
+
+<p class="description">Learn about the typography system and how to customize it.</p>
+
+Joy UI ships a typography system to the theme to help you create consistent texts across your application. You are able to customize the default system or start from scratch depending on your need.
+
+## Default system
+
+It consists of 13 built-in levels that are comprehensive for building a web application.
+
+- The `body1` level is the **baseline** typography for the application (used as a default configuration in [`Typography`](/joy-ui/react-typography/) and [`CssBaseline`](/joy-ui/react-css-baseline/) components).
+- The `body2` to `body5` levels can be used for secondary and tertiary information.
+- The `h1` to `h6` levels follow the semantic HTML headings.
+- The `display1` and `display2` usually appear as taglines for marketing and landing pages.
+
+{{"demo": "DefaultTypographySystem.js"}}
+
+:::success
+**Gotcha**: The [`CssBaseline`](/joy-ui/react-css-baseline/), [`ScopedCssBaseline`](/joy-ui/react-css-baseline/#scoping-on-children) and [`Typography`](/joy-ui/react-typography/) are the only components that consume the theme typography directly.
+
+This ensures that you can customize or even remove the default typography system without affecting other components.
+:::
+
+### Customizing the default system
+
+Provide the key and a CSS object to `theme.typography` to override the default style.
```

Comment:
To make it more clear how this differs from adding new levels.

```suggestion
To customize a default level, provide its name as a key and an object containing CSS rules as a value to `theme.typography`.
```

Reply:
I have to remember this sentence 🤩.